### PR TITLE
Created music_album & music_genre classes

### PR DESCRIPTION
**Pull request by Somdotta07**
- Created `MusicAlbum` class in a separate .rb file.
- Created `Genre` class with an association to the Item class (in a separate .rb file).
- Set up in the constructor method `MusicAlbum` class properties visible in the diagram 
- Set up in the constructor method `Genre` class properties visible in the diagram
- Implemented methods` add_item ` & `can_be_archived?()`

### CODE REVIEW

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Your app is running free of `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: ruby_classes/music_album.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  def initialize(name, publish_date, on_spotify)
+    super(publish_date)
+    @name = name
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super && @on_spotify
+  end
```
Comment (ShahierNashaat) [inline ruby_classes/music_album.rb:16]: Please make the can_be_archived? method private

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]

File: ruby_classes/music_album.rb (hunk 1 above)
Comment (Somdotta07) [reply ruby_classes/music_album.rb:16]: Thanks for your feedback. I'll fix this issue.

**Review by ShahierNashaat — APPROVED** [review]
Hi @Somdotta07  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**